examples/native-external-references-mockshop/src/tools/types.ts

```diff
@@ -0,0 +1,71 @@
+type Link<T> = {
+  sys: {
+    type: string;
+    linkType: T;
+    id: string;
+  };
+};
+
+export type ResourceProvider = {
```

Comment:
@vikaskumr we should be able to re-use `contentful-management` here.

Reply:
🤔

Reply:
@anho I am sure you are talking about this import [here](https://github.com/contentful/contentful-management.js/blob/8bafe383264504405d5f971d721727ff4b600288/lib/entities/resource-provider.ts#L14) in CMA
But I am not able to import this using CMA.

Reply:
Got it, I was trying in old CMA version 😮‍💨